src/Grafana/dashboards/aspnetcore.json

```diff
@@ -1281,16 +1292,17 @@
   "templating": {
     "list": [
       {
+        "allValue": ".*",
```

Comment:
This is the `job` template variable, and this `.*` value as a custom `allValue`  property is actually an important performance optimization for Grafana --> Prometheus. PromQL queries can't be any larger than 64kb or so (I forget the limit, but it's not that large.) If we're using template variable expansion across a large number of unique jobs / server instances (i.e. hundreds or thousands), it's very easy to hit this figure.

Substituting the "all" value with this `.*` wildcard in Prometheus eliminates this issue and keeps the size of the PromQL query small when "all" is selected.